addons/openebs/3.3.0/install.sh

```diff
@@ -49,58 +49,47 @@ function openebs_maybe_migrate_from_rook() {
     if [ -z "$ROOK_VERSION" ]; then
         if kubectl get ns | grep -q rook-ceph; then
             # show validation errors from pvmigrate
-            # if there are errors, maybe_prompt_migrate_from_rook_dryrun_errors() will bail
-            maybe_prompt_migrate_from_rook_dryrun_errors
-            rook_ceph_to_sc_migration "$OPENEBS_LOCALPV_STORAGE_CLASS"
+            # if there are errors, maybe_rook_to_openebs_migration_checks() will bail
+            maybe_rook_to_openebs_migration_checks
+            rook_ceph_to_sc_migration "$OPENEBS_LOCALPV_STORAGE_CLASS" "1"
             DID_MIGRATE_ROOK_PVCS=1 # used to automatically delete rook-ceph if object store data was also migrated
         fi
     fi
 }
 
-function maybe_prompt_migrate_from_rook_dryrun_errors() {
-    echo "Running Rook to OpenEBS Migration Checks ..."
+function maybe_rook_to_openebs_migration_checks() {
```

Comment:
https://github.com/replicatedhq/kURL/blob/main/addons/README.md#requirements

> All shell functions defined in the add-on's install.sh script should be prefixed with the name of the add-on to avoid naming collisions, for example function rook_dashboard_ready_spinner().

```suggestion
function openebs_maybe_rook_to_openebs_migration_checks() {
```

Reply:
I've addressed this.